src/lib389/lib389/topologies.py

```diff
@@ -124,6 +124,7 @@ def _create_instances(topo_dict, suffix):
             instance.use_ldap_uri()
             instance.open()
             instance.config.set('nsslapd-accesslog-logbuffering','off')
+            instance.config.set('nsslapd-auditlog-logbuffering','off')
```

Comment:
But I see that you disable it here in `_create_instances` - so it's not tested. Or do I miss something?

Reply:
What I was joking about was if it's broken when it's absent, but it works when its present, then that's a form of testing.  But I said I knew what you meant, and I will add a test case...

Reply:
Oh, okay! :D Yeah, true)